app/views/buddyships/new.html.erb

```diff
@@ -23,7 +23,7 @@
                   <hr>
                   <p class="text-muted m-t-15"><%= buddy.goal_description %></p>
 
-                  <div class="btn btn-success"><%= link_to "Select", buddyships_path(buddy: buddy.id), method: "POST" %></div>
+                  <div class="btn btn"><%= link_to "Select", buddyships_path(buddy: buddy.id), method: "POST" %></div>
```

Comment:
Juli, does just class="btn" (not "btn btn") work here as well?

Reply:
If we were working only with CSS, btn would work. btn btn is a Bootstrap code.

Reply:
ok